package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "lesson02",
+  "version": "1.0.0",
+  "description": "lesson02 ========",
+  "main": "ArithmeticServer.js",
+  "dependencies": {
```

Comment:
Thanks!
Please also migrate existing files to ts

Reply:
done